api/docker/php/docker-entrypoint.sh

```diff
@@ -36,4 +36,4 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 	fi
 fi
 
-exec docker-php-entrypoint "$@"
+exec docker-php-entrypoint php-fpm
```

Comment:
Doesn't matter anyway since the original script was just passing the php-fpm option in the CMD:

![image](https://user-images.githubusercontent.com/23189983/116199940-fcb9b180-a737-11eb-84a1-ff954b15c30d.png)

When I call the docker-entrypoint.sh script i pass it 2 more variables which are http_proxy and https_proxy. If we let the:

> exec docker-php-entrypoint "$@" it could break other things.

Reply:
Thinking again about that we could simply remove http_proxy and https_proxy from the $@ object. That way we could pass any arguments except the proxy conf. If you're willing to I can modify the bash script acordingly :)

Reply:
Here is what I came with:

```sh
 #!/bin/sh
 
 echo $@
 
 for arg do
     shift
     case "$arg" in
         http_proxy*  ) :                   ;;
         https_proxy* ) :                   ;;
         *            )  set -- "$@" "$arg" ;;
     esac
 done
 
 echo $@
```

Command:

> sh test.sh php-fpm http_proxy=http://0.0.0.0 https_proxys=http://0.0.0.0 whatever
 
Output: 

![image](https://user-images.githubusercontent.com/23189983/116458640-2b817600-a865-11eb-8958-d9afb9ce8e97.png)